services/scan/src/precinct_scanner_app.ts

```diff
@@ -440,13 +453,26 @@ export async function buildPrecinctScannerApp(
 
   app.post<NoParams, Scan.ExportResponse, Scan.ExportRequest>(
     '/precinct-scanner/export',
-    (_request, response) => {
-      const outputStream = new streams.WritableStream();
-      store.exportCvrs(outputStream);
-      const cvrs = outputStream.toString();
+    (request, response) => {
+      const skipImages = request.body?.skipImages;
+      debug(`exporting CVRs ${skipImages ? 'without' : 'with'} inline images`);
+
+      const cvrFilename = generateFilenameForScanningResults(
+        // TODO: Move machine config provider to shared utilities and access
+        // actual machine config, with dev overrides, here instead
+        '0000',
+        getBallotsCounted(store),
+        store.getTestMode(),
+        new Date()
+      );
+
+      response
+        .header('Content-Type', 'text/plain; charset=utf-8')
+        .header('Content-Disposition', `attachment; filename="${cvrFilename}"`);
+      const cvrStream = new PassThrough();
```

Comment:
Why not `store.exportCvrs(response, skipImages)`? Why do we have the `PassThrough` stream?

Reply:
`exportCvrs` is expecting a Writable interface, but we also need it to be a Readable interface to pipe it into the response. The `Passthrough` just gives us both. Do you know an easier way?

Reply:
The `response` is writable. I just changed it to this and it appears to work with kiosk-browser and pass the tests.

Reply:
Oh, that makes a lot of sense. Will change to be so.

Reply:
Had to use Passthrough here, otherwise it didn't work!